games/minimal/mods/default/init.lua

```diff
@@ -730,8 +730,10 @@ end
 
 -- Register nodes
 
+local S = core.get_translator("test")
+
 minetest.register_node("default:stone", {
-	description = "Stone",
+	description = S("Stone @1", minetest.colorize("red", "arg")),
```

Comment:
Yes, of course. This is only to show what happens when an argument is used, and to demonstrate that arguments can contain escape sequences as well.